lib/krane/deprecated_deploy_task.rb

```diff
@@ -447,6 +446,33 @@ def deploy_resources(resources, prune: false, verify:, record_summary: true)
       end
     end
 
+    def replace_or_create_resource(resource, force: false)
+      args = if force
+        ["replace", "--force", "--cascade", "-f", resource.file_path]
+      else
+        ["replace", "-f", resource.file_path]
+      end
+
+      _, _, status = kubectl.run(*args, log_failure: false)
```

Comment:
We should also pass `output_is_sensitive: resource.sensitive_template_content?` in this version (we didn't before, but that's great fix you've made below!)

Reply:
Good eye, I missed this here.